metrics/Code_Changes_Lines.md

```diff
@@ -1,7 +1,10 @@
 # Code Changes Lines
-Question: What is the sum of the number of lines touched (lines added plus lines removed) in all changes to the source code during a certain period?
+
+**Question:** What is the sum of the number of lines touched (lines added plus lines removed) in all changes to the source code during a certain period?
```

Comment:
Don't bold the question

Reply:
It was bolded on the code changes metric. We see its not bolded on the template. We can remove that.